### ISSUE TITLE
Folder note feature stopped working (6.3.4)

### ISSUE THREAD

**Issue by hdferguson3** [openkm/document-management-system#47]
Lost ability to add notes to category folders.

The "add" button is now greyed out and the text box is missing.

![screenshot from 2017-12-04 09-24-19](https://user-images.githubusercontent.com/34278872/33616030-aaaf10b0-d9a9-11e7-950f-fbaf0db9b110.png)

**Comment by monkiki** [comment]
Can you elaborate the issue? Was it working and now stopped? 

I've checked with 6.3.5-DEV and works fine. We're planning to make a release soon.

Regards.

**Comment by hdferguson3** [comment]
Yes, I was able to previously add notes to a categories folder. The
attached image from this post
https://forum.openkm.com/viewtopic.php?f=5&p=45014#p44991 shows that I was
able to add a note on 28-9-2017 14:29:09. Yet now when I try to add notes
to a categories folder the entry box is not showing and the "Add" button is
greyed out.

**Comment by monkiki** [comment]
Well I can't understand why you were able to add notes and right now you can't. Have you changed something in the source code, o in the node security? As I said, I've checked in 6.3.5-DEV and works fine. And no other user has had this issue, so it's a bit weird.

**Comment by hdferguson3** [comment]
I have not changed anything with the source code. Is there anything simple
that I might be over looking with regards to Category folders? Like I said,
I can use the Notes feature on Taxonomy and My Documents folders, just not
Category folders. Since the "Add" button is still there I expect it is some
configuration setting I am over looking. I have OpenKM in a VM, I will fire
it up and see if I can still use the feature there and do an investigation.
But any configuration settings that you can think of for me to check would
be greatly appreciated.

**Comment by monkiki** [comment]
Sorry I didn't realise the screenshot was referring to categories. In fact, you can't add notes to categories. Maybe this can be implemented but nobody else has requested this feature.

**Comment by hdferguson3** [comment]
So, how dd I attach a note to the category folder shown in the screenshot?

**Comment by monkiki** [comment]
Maybe using the API? Or maybe the button was not disabled due to a Javascript glitch.

**Comment by hdferguson3** [comment]
Ok, I suppose the wiki can serve the same purpose of tracking information at a Category level. Thanks for looking into this.
